### ISSUE TITLE
[beta] Create a Media Query - Instructions and test don't match

### ISSUE THREAD

**Issue by webdevjeffus** [freeCodeCamp/freeCodeCamp#13264]
Challenge [create-a-media-query](http://beta.freecodecamp.com/en/challenges/responsive-web-design-principles/create-a-media-query) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.95 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```html


<style>
  @media (max-height: 800px) {
    p { font-size: 10px }
  }
</style>

<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus quis tempus massa. Aenean erat nisl, gravida vel vestibulum cursus, interdum sit amet lectus. Sed sit amet quam nibh. Suspendisse quis tincidunt nulla. In hac habitasse platea dictumst. Ut sit amet pretium nisl. Vivamus vel mi sem. Aenean sit amet consectetur sem. Suspendisse pretium, purus et gravida consequat, nunc ligula ultricies diam, at aliquet velit libero a dui.</p>

```
In the "Create a Media Query" challenge, the instructions and the test don't match. The instructions say:

> Add a media query, so that the p tag has a font-size of 10px when the device's height is smaller than 800px.

**Smaller than 800px** means 799 or less, so the correct media query should be "max-width: 799px". The test actually passes with "max-width: 800px", which translates into English as **800px or smaller**.

Basically, the instructions call for **width < 800**, but the test checks for **width <= 800**. Either one is fine, but they need to match each other.

**Comment by cdrainxv** [comment]
I agree with the revision suggested above - smaller than denotes the exclusion of 800px. A simple fix would be:

Instruction:
> Add a media query so that the p element has a font-size of 10px when the device's height is less than or equal to 800px.

Tests:

> 1. Your p element should have the font-size of 10px when the device height is less than or equal to 800px.
> 2. Declare a @media query for devices with heights less than or equal to 800px.


Another change I want to propose is to really make it evident that there has been a style change by way of the media query. This can be done by simply including a font-size property for the p element in the challenge code, then when the camper adds the media query its purpose and its effect will be rather self-evident. 

This is my proposal for the challenge code:

```
<style>
  p {
    font-size: 20px;
  }
  /* Add media query below */

</style>

<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus quis tempus massa. Aenean erat nisl, gravida vel vestibulum cursus, interdum sit amet lectus. Sed sit amet quam nibh. Suspendisse quis tincidunt nulla. In hac habitasse platea dictumst. Ut sit amet pretium nisl. Vivamus vel mi sem. Aenean sit amet consectetur sem. Suspendisse pretium, purus et gravida consequat, nunc ligula ultricies diam, at aliquet velit libero a dui.</p>
```

Thanks!

**Comment by HKuz** [comment]
Thanks for the issue @webdevjeffus and the proposed solution @cdrainxv - agree with both of you this should be consistent to avoid confusion. @cdrainxv your suggestion for a fix nails it, and making it more obvious of the text size change sounds good, so let's go with that 👍 

Contributors - If you'd like to help make these changes, please check out the [Contributor's Guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md) and ask any questions in the [Contributor's Chat Room](https://gitter.im/FreeCodeCamp/Contributors). The code for this challenge can be found [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/seed/challenges/01-responsive-web-design/responsive-web-design.json#L57). Thank you! 🎉 🌵

**Comment by rrjoson** [comment]
I'll take this!

**Comment by cdrainxv** [comment]
@rrjoson: Oh, I had already started on this, but alas I forgot to call dibs. :smile: Nothing is holier than a dib - good luck! Thanks for contributing!

**Comment by rrjoson** [comment]
Oh, sorry about that! Will ask before I work on an issue next time! 😄

**Comment by wuxmedia** [comment]
only because this is the first result for googling this error - check the forum post - basically you need to make sure Chrome's font sizes are 10px minimum (otherwise this just doesn't pass ever)
